apps/src/applab/applab.js

```diff
@@ -44,6 +44,7 @@ var showAssetManager = require('./assetManagement/show.js');
 var DebugArea = require('./DebugArea');
 var VisualizationOverlay = require('./VisualizationOverlay');
 var ShareWarningsDialog = require('../templates/ShareWarningsDialog.jsx');
+var newrelic = require('../newrelic');
```

Comment:
If you forget to include this, the call to `newrelic.addPageAction` is still going to "work", but without the protections you've added. I would suggest calling the var and the newrelic filename (or the function name) something slightly different to prevent this from happening by accident.